projects/laji/src/app/shared-modules/observation-map/observation-map/observation-map.component.ts

```diff
@@ -49,19 +50,12 @@ interface AggregateQueryResponse {
 }
 
 // Given coordinates in warehouse query format
-// Returns a featureCollection visualizing that set of coordinates
-const getFeatureCollectionFromQueryCoordinates$ = (coordinates: any): Observable<any> => (
+// Returns features visualizing that set of coordinates
+const getFeaturesFromQueryCoordinates$ = (coordinates: string[]): Observable<G.Feature<G.Polygon>[]> => (
   ObservableOf(coordinates
     ? coordinates.map(
-      (coord: any) => getFeatureFromGeometry(convertLajiEtlCoordinatesToGeometry(coord))
+      (coord) => getFeatureFromGeometry(convertLajiEtlCoordinatesToGeometry(coord))
```

Comment:
If we had `strict` typing enabled, tsc would complain about implicit any, so we should probably keep the type in there.